services/graph/pkg/service/v0/sharedwithme_test.go

```diff
@@ -177,12 +161,18 @@ var _ = Describe("SharedWithMe", func() {
 			Expect(jsonData.Get("#").Num).To(Equal(float64(2)))
 		})
 
-		It("populates the driveItem properties", func() {
+		// TODO clarify which of the various properties should actually be set on the driveItem and what
+		//      semantics those properties have
+		// createdDateTime: is this the share creation time or the creation time of the shared item (there is not ctime in the stat response)?
+		// lastModifiedDateTime: see above
+		// owner: Should this be the owner of the shared item, the creator of the share, or the owner of the sharejail?
+		// etag: should this just equal the etag of the inner remote item?
```

Comment:
This still needs clarification. Currently the properties of the outer driveItem is more or less a duplication of the inner remoteItem. Apart from the redundancy, is that really correct? Shouldn't we avoid all that duplication?

We need to properly define what info belongs where. Otherwise we'll just get mess.

@micbar @butonic  can you help with that.